docs/adrs/0262-custom-label-support.md

```diff
@@ -0,0 +1,43 @@
+# ADR 262: Custom Label Support
```

Comment:
rename this and file to 275.  ADR number is PR number so easy to find and no collisions. :)